### DISCUSSION TITLE
鼠标默认操作没了

### DISCUSSION THREAD

**Discussion by xiaoyanxyxy** [ywmoyue/biliuwp-lite#901] [bug]
### Bug 描述

之前设置的是：
- 双击暂停 单击无操作
- 可左右滑动代替手势后退前进
- 可长按屏幕倍数播放
现在新版君已失效，变成了：
- 双击全屏 单击暂停播放
- 已无左右滑动前进后退
- 已无长按倍数播放

### 操作步骤

```plain text
直接用鼠标操作现在只有一个单击着一个功能了，滑动和长按均无识别 ，平板电脑更是难以操控
```

### 期望结果

期望基于鼠标完整快捷设置，并提供最高操作优先级，可以考虑引入edge的手势操作，“↓→”关闭标签太好用了，
期望能提供基于平板电脑完整的手势操作，经常用平板，关闭标签太难点按了，现在平板上手势操作也失效了，

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

24H2

### 设备类型

台式机, 平板

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
请用回正式版，最新临时版的鼠标操作不会是正式版的默认操作

**Comment by ProJend** [comment]
已有的操作会保留的，这已经不是简单修改的问题了，想要找到触摸和鼠标之间的平衡点就要重写底层代码的操作逻辑
